GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -54,6 +54,19 @@ public enum IgnoreSubmodulesMode
         All = 5
     }
 
+    /// <summary>Mode for 'git clean'</summary>
+    public enum CleanMode
+    {
+        /// <summary>Only untracked files not in .gitignore, the default</summary>
+        OnlyNonIgnored = 1,
+
+        /// <summary>Only files included in .gitignore</summary>
```

Comment:
```suggestion
        /// <summary>Only files included in any ignore list(.gitignore, $GIT_DIR/info/exclude).  Git clean with -X option.</summary>
```

Reply:
Updating (not committing, double space after period)